Accounts: Add caveats about consumption (#14913)
newrelic/docs-website: src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/usage-queries-alerts.mdx

freshnessValidatedDate: never
---

To help you understand your [New Relic usage and billing](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing), we provide a [usage UI](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing#usage-ui) for understanding your billing-related usage. Also, you can run custom NRQL queries to get additional detail, and you can create alert notifications for when your usage has reached certain levels. 

Keep the following in mind when using the usage UI or making consumption queries:

* The results are approximate and don't correspond exactly to the values used to calculate your invoice amount. 

> * Use these tools to find and track areas of high usage.

Reviewer: ```suggestion
* Use these tools to find and track your areas of high usage.
```

* Use queries to create alerts so you know if you're approaching a limit.

## Data ingest usage queries [#data-queries]

The [usage UI](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing#usage-ui) displays your data ingest over time. If you need more detail than the UI provides, you can use a variety of NRQL queries. The examples below include two data types:

> * `NrConsumption`: This is useful for diving deep into the hourly details of usage. It's less helpful as a proxy for your bill.

Reviewer: ```suggestion
* `NrConsumption`: Use this to dive deep into the details of your usage hour-by-hour. It's less helpful as a proxy for your bill.
```

How about this?

> * `NrMTDConsumption`: This has a monthly aggregation of usage that gives you a good approximation of usage.

Reviewer: ```suggestion
* `NrMTDConsumption`: Use this to see a good approximation of your monthly aggregated usage.
```

How about this?
